app/helpers/pets_helper.rb

```diff
@@ -1,2 +1,11 @@
 module PetsHelper
+  def status_badge(pet)
+    if pet.has_adoption_pending?
+      BadgeComponent.new('Adoption Pending', class: 'badge bg-info')
+    elsif pet.is_adopted?
+      BadgeComponent.new('Adopted', class: 'badge bg-warning')
+    else
+      BadgeComponent.new('Free', class: 'badge bg-light-warning')
```

Comment:
I think the bg-light-warning class for the open status isn’t very readable.

![image](https://github.com/user-attachments/assets/114bc2f1-27c5-4ace-a168-a5ab4fb2ad57)

Reply:
Yes you are right. This is not accessible. Perhaps we could have black text on the background? Does that look better?